parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -65,13 +68,19 @@ installEtcd() {
 
 installDeps() {
     if [[ $OS == $UBUNTU_OS_NAME ]]; then
-      CEPH_COMMON="ceph-common"
-      GLUSTERFS_CLIENT="glusterfs-client"
+        if [[ $UBUNTU_VERSION == "18.04" ]]; then
+            NTP="ntp"
+            systemctl_stop 20 5 10 systemd-timesyncd || exit $ERR_SYSTEMCTL_STOP_FAIL
+            retrycmd_if_failure 120 5 25 systemctl disable systemd-timesyncd || exit $ERR_SYSTEMCTL_STOP_FAIL
```

Comment:
can we put disabling timesyncd in a separate function since it's not actually part of installing any dependencies?

Reply:
I would prefer not to install the ntp package on an OS runtime w/ systemd-timesyncd running and enabled, but maybe that's paranoia?

Reply:
we can at least extract it into a func and call it from here, just want to make sure installDeps doesn't become an long list of unrelated commands